### PR TITLE
Only Display Freemius to 10% to New Customers

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

File: src/Tribe/Integrations/Freemius.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function __construct() {
 		 *
 		 * @param  bool  $should_load
 		 */
-		$should_load = apply_filters( 'tribe_events_integrations_should_load_freemius', true );
+		$should_load = apply_filters( 'tribe_events_integrations_should_load_freemius', $this->should_load( 10 ) );
```
Comment (Camwyn) [inline src/Tribe/Integrations/Freemius.php]: Do we need to pass the default value here?

File: src/Tribe/Integrations/Freemius.php [hunk 2]
```diff
@@ -96,6 +96,51 @@ public function __construct() {
...
+	 * When should we load Freemius to users
+	 *
+	 * @since  TBD
+	 *
+	 * @param  integer $threshold Percentage of which we will load Freemius
+	 *
+	 * @return boolean
+	 */
+	public function should_load( $threshold = 10 ) {
+		$previous_versions = Tribe__Settings_Manager::get_option( 'previous_ecp_versions', [] );
```
Comment (juanfra) [inline src/Tribe/Integrations/Freemius.php]: is this checking for ECP or TEC?

Comment (bordoni) [reply src/Tribe/Integrations/Freemius.php]: That was the string that was set way back when it was created. But it's right

File: src/Tribe/Integrations/Freemius.php (hunk 1 above)
Comment (bordoni) [reply src/Tribe/Integrations/Freemius.php]: Just for future reference, we will change that int there in the future.

**Pull request merged**